app/controllers/peoplefinder/people_controller.rb

```diff
@@ -14,6 +14,7 @@ def index
 
     # GET /people/1
     def show
+      @versions = AuditVersionPresenter.wrap(@person.versions)
```

Comment:
Can we only load this when the super admin is logged in, rather than checking in the view?